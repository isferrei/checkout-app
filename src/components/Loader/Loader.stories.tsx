import type { Meta, StoryObj } from "@storybook/react";

import { Loader } from "./Loader";

const meta = {
  title: "Example/Loader",
  component: Loader,
  tags: ["autodocs"],
  parameters: {
    layout: "fullscreen",
  },
} satisfies Meta;

export default meta;
type Story = StoryObj;

export const Default = {
  args: {},
} satisfies Story;
