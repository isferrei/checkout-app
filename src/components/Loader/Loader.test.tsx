import { render } from "@testing-library/react"
import React from "react"
import { Loader } from "./Loader"

describe("Loader component", () => {
    it("should render correctly", () => {
        const { container } = render(<Loader />)
        expect(container).toMatchSnapshot()
    })
})
