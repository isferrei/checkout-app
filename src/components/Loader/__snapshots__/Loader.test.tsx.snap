// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component should render correctly 1`] = `
<div>
  <div
    class="flex justify-center items-center h-screen"
  >
    <div
      class="mx-auto inline-block h-8 w-8 animate-spin rounded-full border-4 border-blue border-solid border-current border-r-transparent align-[-0.125em] motion-reduce:animate-[spin_1.5s_linear_infinite]"
      role="status"
    />
  </div>
</div>
`;
